src/components/Form/SelectField.tsx

```diff
@@ -0,0 +1,54 @@
+import { ReactNode } from "react";
+import Box from "@mui/material/Box";
+
+//SELECT
+import TextField from "@mui/material/TextField";
+import MenuItem from "@mui/material/MenuItem";
+
+type SelectProps = {
+  selectOptions: string[];
```

Comment:
It's possible that you will sometimes have to separate the value and name of the select field's options. E.g. you will want to display a select option to the user that says "Whole house" but you'd have to send the value "house" or "wholeHouse" or similar to the backend.

So consider having the type of selectOptions as  
```ts
selectOptions: { name: string; value: string; }[];
```  
Then you'll have an array of these options to iterate and use